Source/WebKit/UIProcess/Cocoa/PlaybackSessionManagerProxy.h

```diff
@@ -38,13 +38,17 @@
 #include <wtf/RefCounted.h>
 #include <wtf/RefPtr.h>
 
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(WATCHOS)
+#include <WebCore/NullPlaybackSessionInterface.h>
+#elif PLATFORM(IOS_FAMILY)
 #include <WebCore/PlaybackSessionInterfaceAVKit.h>
 #else
 #include <WebCore/PlaybackSessionInterfaceMac.h>
 #endif
 
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(WATCHOS)
+typedef WebCore::NullPlaybackSessionInterface PlatformPlaybackSessionInterface;
```

Comment:
I know you're just copying the pattern used on other platforms, but would a forward declaration of `NullPlaybackSessionInterface` be sufficient here to define the typedef?